add_user.php

```diff
@@ -64,8 +64,8 @@
 										//creates a variable to store sql code for the 'add users' query which will add the user's details (username and password) to my Users table in phpMyAdmin
 										$insertquery = "INSERT INTO Users( User_PK, Password ) VALUES( '$UserID', '$PW' )";
 									?>
-									<center>
-										<img style="padding-top: 5px; width: 100%; max-width: 200px;" src="images/add-icon.png" alt="Profile graphic">
+									<div class="center" style="text-align: center;">
```

Comment:
#replaced all <center> html tags with a div slass to represent the same CSS properties